Package.swift

```diff
@@ -6,8 +6,7 @@ import PackageDescription
 let package = Package(
     name: "CodeScanner",
     platforms: [
-      .iOS(.v13),
-      .macOS(.v11)
```

Comment:
Are you sure there is not something missing for catalyst?

Reply:
The package builds fine for Catalyst, but not for macOS. To add a minimum deployment target for Catalyst, we would need to update the Swift version, which is 5.3. This does not support a .macCatalyst(_:) option.